pkg/xds/generator/inbound_proxy_generator.go

```diff
@@ -103,3 +116,44 @@ func (g InboundProxyGenerator) Generate(ctx xds_context.Context, proxy *model.Pr
 	}
 	return resources, nil
 }
+
+func (g *InboundProxyGenerator) BuildInboundRoutes(cluster envoy_common.Cluster, rateLimits []*mesh_proto.RateLimit) (envoy_common.Routes, error) {
+	routes := envoy_common.Routes{}
+
+	for _, rateLimit := range rateLimits {
+		if rateLimit.GetConf().GetHttp() != nil {
+			route := envoy_common.NewRouteFromCluster(cluster)
+			// Source
```

Comment:
`// source`?

Reply:
I'd leave a comment somewhere around this function that we rely on the fact that rateLimits are in order from most specific to most general